### PR TITLE
[RPM] Update dynflow to 0.8.37

### PR DESCRIPTION
**Pull request by adamruzicka**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
http://koji.katello.org/koji/taskinfo?taskID=87036

File: packages/foreman/rubygem-dynflow/rubygem-dynflow.spec [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 Summary: DYNamic workFLOW engine
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.8.36
+Version: 0.8.37
 Release: 2%{?foremandist}%{?dist}
```
Comment (ekohl) [inline packages/foreman/rubygem-dynflow/rubygem-dynflow.spec]: Didn't notice the release which should have been set to 1 before releasing :(

Comment (adamruzicka) [reply packages/foreman/rubygem-dynflow/rubygem-dynflow.spec]: Oh, neither did I. Will we try to fix it somehow or will we let it slide?

Comment (ekohl) [reply packages/foreman/rubygem-dynflow/rubygem-dynflow.spec]: Doesn't hurt and we should try to fix it at the next release.

Comment (adamruzicka) [reply packages/foreman/rubygem-dynflow/rubygem-dynflow.spec]: Ok, adding it to my checklist so I won't forget it

**Pull request merged**